mender-convert-package

```diff
@@ -95,11 +95,11 @@ alignment_sectors=$((${MENDER_PARTITION_ALIGNMENT} / 512))
 
 if [ "${MENDER_GRUB_EFI_INTEGRATION}" == "y" ]; then
   boot_part_start=${alignment_sectors}
-  overhead_sectors=$(( ${alignment_sectors} * 4 ))
```

Comment:
You have a lot of these changes where you are removing "whitespaces" and would try no to mix this with indentation fixes.

Also in this particular case I vote to have extra white space before/after for `$((  ))` statements for readability purposes. This probably applies to many other locations and would simple exude these changes for now

Reply:
Alright, fixing up